src/services/chat.js

```
@@ -0,0 +1,57 @@
+export async function POST(request) {
+  try {
+    const { message, history } = await request.json();
+
+    // Call Deepseek API
+    const response = await fetch(
+      "https://api.deepseek.com/v1/chat/completions",
+      {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+          Authorization: `Bearer ${process.env.DEEPSEEK_API_KEY ?? ""}`,
```

ESLint

[error] 'process' is not defined.

(no-undef)